### ISSUE TITLE
"isArray is not defined"

### ISSUE THREAD

**Issue by JuliusGudni** [VirtusLab-Open-Source/strapi-plugin-navigation#159]
The following error happens on a clean Strapi v.4.1.0 installation.
Was able to replicate the error in all usage scenarios, as in with the UI Navigation completely empty, not-empty both with relations, no relations, nesting, no nesting.

Looked at the code and there seems to be a isArray import missing from loadash. Will be making a pull request in a few.

```
 [2022-02-21 22:25:28.076] error: isArray is not defined
ReferenceError: isArray is not defined
    at Object.composeItemTitle (C:\Users\Extis\Documents\admin-v4-ssfm.is\node_modules\strapi-plugin-navigation\server\services\utils\functions.js:192:36)
```

**Comment by JuliusGudni** [comment]
#160

**Comment by cyp3rius** [comment]
@JuliusGudni thank you for the spot and fix. Released as part of `v2.0.4`
